RSDKv5/RSDK/Dev/devfont_to_hpp.py

```diff
@@ -0,0 +1,74 @@
+# Requires the Pillow dependency
+import sys, os, math, itertools
+from PIL import Image
+from pathlib import Path
+
+WIDTH = 8
+HEIGHT = 8
+CHARCOUNT = 128
+COLCOUNT = 16
+
+IMGWIDTH = (COLCOUNT if CHARCOUNT >= COLCOUNT else CHARCOUNT) * WIDTH
+IMGHEIGHT = math.ceil(CHARCOUNT / COLCOUNT) * HEIGHT
+
+def encode_font(image_path):
+    print(f"Opening {image_path}...")
+
+    img = Image.open(image_path).convert("1")
+    iwidth, iheight = img.size
+    if iwidth != IMGWIDTH or iheight != IMGHEIGHT:
+        print(f"[ERROR] Image does not match expected size ({IMGWIDTH}x{IMGHEIGHT})")
+        return []
```

Comment:
Fatal error can be handled with assert instead:
```suggestion
    assert img.size == (IMGWIDTH, IMGHEIGHT), f"Image does not match expected size ({IMGWIDTH}x{IMGHEIGHT})"
```